### PR TITLE
Replace the `filename` tag with the `literal` tag for the `.php` file extension

### PR DESCRIPTION
**Pull request by mmalferov**

### CODE REVIEW

**Comment by mmalferov** [comment]
I also removed the trailing spaces that lead to linter errors in [install/macos/bundled.xml](https://github.com/php/doc-en/pull/3870/files#diff-39191bf405c0d2ce6613e2611f867c14d7fb02a3b65fbfe2b1a2337805f91ecb)

File: chapters/tutorial.xml [hunk 1]
```diff
@@ -20,11 +20,11 @@
    <info><title>What do I need?</title></info>
    <para>
     In this tutorial we assume that your server has activated support 
-    for PHP and that all files ending in <filename>.php</filename>
```
Comment (Girgias) [inline chapters/tutorial.xml:23]: Those should be `<filename type="extension">` and not `<literal>`. See: https://tdg.docbook.org/tdg/5.2/filename

**Review by Girgias — CHANGES_REQUESTED** [review]

File: chapters/tutorial.xml (hunk 1 above)
Comment (mmalferov) [reply chapters/tutorial.xml:23]: It raise an error:

```
ERROR (file:////home/runner/work/doc-en/doc-en/en/chapters/tutorial.xml:23:68)
    for PHP and that all files ending in <filename type="extension">.php</filename>
--------------------------------------------------------------------^
 No declaration for attribute type of element filename
```

Comment (cmb69) [reply chapters/tutorial.xml:23]: Ah, thank you!

Comment (Girgias) [reply chapters/tutorial.xml:23]: Ah my bad it is a `class` attribute and not a `type` attribute. (DocBook and the very consistent attribute names)

Comment (mmalferov) [reply chapters/tutorial.xml:23]: Bring back the `literal` tag for the `.php` extension?

Comment (mmalferov) [reply chapters/tutorial.xml:23]: Done

File: reference/opcache/preload.xml [hunk 2]
```diff
@@ -53,7 +53,7 @@ opcache.preload=preload.php
   In the <filename>preload.php</filename> script, any file referenced by <function>include</function>,
   <function>include_once</function>, <function>require</function>, <function>require_once</function>, or
   <function>opcache_compile_file</function> will be parsed into persistent memory.  In the following example,
-  all <filename>.php</filename> files in the <filename>src</filename> directory will be preloaded, unless they
+  all <filename class="extension">.php</filename> files in the <filename>src</filename> directory will be preloaded, unless they
```
Comment (Girgias) [inline reference/opcache/preload.xml:56]: This is more for a follow-up PR, but directories should ideally also have the corresponding `class` attribute. :)
```suggestion
  all <filename class="extension">.php</filename> files in the <filename class="directory">src</filename> directory will be preloaded, unless they
```

File: install/macos/bundled.xml [hunk 3]
```diff
@@ -167,23 +168,23 @@
...
      <filename>http://localhost/~yourusername/info.php</filename>
     </para>
    </listitem>
   </orderedlist>
  </para>
  <simpara>
-  The <acronym>CLI</acronym> (or <acronym>CGI</acronym> in older versions) is 
-  appropriately named <filename>php</filename> and likely exists as  
+  The <acronym>CLI</acronym> (or <acronym>CGI</acronym> in older versions) is
+  appropriately named <filename>php</filename> and likely exists as
```
Comment (Girgias) [inline install/macos/bundled.xml:184]: This feels like it should use a `command` tag, but again follow-up PR :)
```suggestion
  appropriately named <command>php</command> and likely exists as
```

**Review by Girgias — COMMENTED** [review]
I added some suggestions for follow-up PRs.

**Review by Girgias — APPROVED** [review]
Thank you!

**Pull request merged**